2-Browsers/Week1/homework/ex1-bookList/index.js

```diff
@@ -31,9 +31,7 @@ class DomElement {
   }
   addChild(child) {
     if (Array.isArray(child)) {
-      child.forEach((element) => {
-        this.child.push(element);
-      });
+      this.child = [...child];
```

Comment:
This is not the same as the forEach statement. What if there already is a child there?

Reply:
Could you help me further?

Reply:
I got it.😊

Reply:
Thank you so much, You mentioned a very good point.🙏

Reply:
`this.child = this.child.concat([...child]);`
This solves my fault.👍